tests/e2e/serial/main_test.go

```diff
@@ -232,7 +232,7 @@ func TestSuiteScan(t *testing.T) {
 				},
 			},
 			ID:       "xccdf_org.ssgproject.content_rule_coreos_vsyscall_kernel_argument",
-			Status:   compv1alpha1.CheckResultInfo,
+			Status:   compv1alpha1.CheckResultFail,
```

Comment:
Does this need to be pulled out into it's own PR since https://github.com/ComplianceAsCode/content/pull/11329 landed?

Reply:
https://github.com/ComplianceAsCode/compliance-operator/pull/490 create a PR here